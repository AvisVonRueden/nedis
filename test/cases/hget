*4
$4
HSET
$6
user:0
$4
name
$2
tj
-----------
*4
$4
HSET
$6
user:0
$5
email
$11
tj@test.com
-----------
*3
$4
HGET
$6
user:0
$4
name
-----------
*3
$4
HGET
$6
user:0
$5
email
-----------
*3
$4
HGET
$6
user:1
$5
email
-----------
*3
$3
SET
$3
foo
$3
bar
-----------
*3
$4
HGET
$3
foo
$5
email
