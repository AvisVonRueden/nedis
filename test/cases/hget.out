:1
:1
$2
tj
$11
tj@test.com
$-1
+OK
-ERR Operation against a key holding the wrong kind of value
